tests/unit/lib/thrift_pool_tests.py

```diff
@@ -70,14 +69,24 @@ def test_max_connection_attempts_works_through_config(self, new_retry_policy):
         new_retry_policy.assert_called_with(attempts=5)
 
 
+class ThriftFIFOPoolTests(unittest.TestCase):
+    def test_fifo_queue_config(self):
+        config = {
+            "fifo-test.endpoint": "127.0.0.1:1234",
+            "fifo-test.fifo_queue": True,
+        }
+        pool = thrift_pool.thrift_pool_from_config(config, prefix="fifo-test.")
+        self.assertTrue(isinstance(pool.queue_cls, gevent_queue.Queue))
+
+
 class ThriftConnectionPoolTests(unittest.TestCase):
     def setUp(self):
-        self.mock_queue = mock.Mock(spec=queue.Queue)
+        self.mock_queue = mock.Mock(spec=gevent_queue.Queue)
```

Comment:
This conflicts with https://github.com/reddit/baseplate.py/pull/656/files#diff-3de570a9b43ce8e7774ac672c91e3f49f999a4d88310d9afb12bd997a840a6fdR299. What'd be the best way to resolve it?

cc @RoganMurley

Reply:
I could wait till #656 is merged and rebase this.

Reply:
I reverted my change here.